apps/riscv-tests/isa/rv64uv/vl1r.c

```diff
@@ -2,49 +2,435 @@
 // Solderpad Hardware License, Version 0.51, see LICENSE for details.
 // SPDX-License-Identifier: SHL-0.51
 //
-// Author: Matheus Cavalcante <matheusd@iis.ee.ethz.ch>
-//         Basile Bougenot <bbougenot@student.ethz.ch>
+// Author: Matteo Perotti
 
 #include "vector_macros.h"
-#include <inttypes.h>
-#include <stdbool.h>
-#include <stdint.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
 
-static volatile uint8_t OUTPUT[256] __attribute__((aligned(64)));
+uint64_t counter;
 
+// Vectors are statically allocated not to exceed the stack and go in the UART
+// address space
+
+uint8_t gold_vec_8b[4096];
```

Comment:
From where did you take those vector lengths? Are they part of the `configuration` variables? If so, you should use those variables here as well.

Reply:
Yes, fixing it.